docker-jans-monolith/jans-mysql-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - MYSQL_PASSWORD=1t5Fin3#security
       - MYSQL_ROOT_PASSWORD=1t5Fin3#security
   jans:
-    image: ghcr.io/janssenproject/jans/monolith:1.0.19_dev
+    image: ghcr.io/janssenproject/jans/monolith:1.0.20-1
```

Comment:
The monolith image is mostly used for quickstart/testing bleeding-edge version of Janssen. In my opinion, it's better to use `:1.0.20_dev` instead.

Reply:
done.